routers.js

```diff
@@ -0,0 +1,104 @@
+const express = require("express");
+
+const db = require("./data/db.js");
+
+const router = express.Router();
+
+router.get("/", async (req, res) => {
+  try {
+    const posts = await db.find();
+    res.status(200).json(posts);
+  } catch (error) {
+    console.log(error);
```

Comment:
not a bad idea to still log the error to the console since the `message` doesn't actually include it